dev.tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "include": ["packages/**/*", "typings/**/*"],
+  "compilerOptions": {
```

Comment:
Can we not extend our `package.tsconfig.json`? Ideally we'd be able to do that and keep our compiler options defined in there consistent with here.